src/interactive-window/interactiveWindow.node.ts

```diff
@@ -186,7 +173,7 @@ export class InteractiveWindow implements IInteractiveWindowLoadable {
             };
             // When connecting, we need to update the sys info message
             this.updateSysInfoMessage(this.getSysInfoMessage(metadata, SysInfoReason.Start), false, sysInfoCell);
-            const kernel = await connectToKernel(
+            const kernel = await KernelConnector.connectToKernel(
```

Comment:
I moved all the loose functions for connecting to a kernel into a new static class - KernelConnector.